pkg/cloudvolumes/get.go

```diff
@@ -0,0 +1,123 @@
+// (C) Copyright 2019 Hewlett Packard Enterprise Development LP.
```

Comment:
looks like it would be good to separate the "client" logic from the command logic.
e.g. basically create a "nimble client" that would wrap the http handling and use that nimble client here.
True for get and login